build.gradle.kts

```diff
@@ -123,18 +125,18 @@ val gitVersion: String by lazy {
 }
 
 buildConfig {
-    appName = "mongo-kafka"
-    version = gitVersion
-    clsName = "Versions"
-    packageName = "com.mongodb.kafka.connect"
+    className("Versions")
+    packageName("com.mongodb.kafka.connect")
+    useJavaOutput()
+    buildConfigField("String", "NAME", "\"mongo-kafka\"")
+    buildConfigField("String", "VERSION", provider { "\"${gitVersion}\"" })
 }
 
 /*
  * Testing
  */
 sourceSets.create("integrationTest") {
     java.srcDir("src/integrationTest/java")
-    resources.srcDir("src/integrationTest/resources")
```

Comment:
This is already copied and the build errored with a duplicate copy message so removed.